PKPD/model/model.py

```diff
@@ -0,0 +1,83 @@
+from PKPD.model.abstractModel import AbstractModel
+import myokit
+
+class Model(AbstractModel):
+    def read_mmt_file(self):
+        # calls self.mmtfile to read
```

Comment:
write docstrings for all functions, please.

Reply:
more important for public methods but it's nice to have a short description for all methods.